File: note_oc/CollectionViewCell.m
//
//  CollectionViewCell.m
//  note_oc
//
//  Created by hryan on 16/2/1.
//  Copyright © 2016年 fe. All rights reserved.
//

#import "CollectionViewCell.h"

@interface CollectionViewCell ()




@end


@implementation CollectionViewCell

- (instancetype)initWithCoder:(NSCoder *)aDecoder {
    self = [super initWithCoder:aDecoder];

    return self;
}

- (instancetype)initWithFrame:(CGRect)frame {
    self =[super initWithFrame:frame];
    return self;
}

- (instancetype)init {
    self = [super init];
//    DLog(@"13")
    return  self;
}

- (void)willRemoveSubview:(UIView *)subview {
   
}

- (UIImageView *)imageView {
    if (!_imageView) {
        _imageView = [[UIImageView alloc]initWithImage:[UIImage imageNamed:@"zrx4.jpg"]];
        _imageView.frame = self.contentView.frame;
    }
    return _imageView;
}

- (void)didAddSubview:(UIView *)subview {
    
    [super didAddSubview:subview];
    
    DLog(@"14")
  
    [self.contentView addSubview:self.imageView];
   
}



@end

File: note_oc/CollectionViewCell.swift
//
//  CollectionViewCell.swift
//  laboratory
//
//  Created by hryan on 16/3/10.
//  Copyright © 2016年 fe. All rights reserved.
//

import UIKit

class CollectionViewCell: UICollectionViewCell {
    var label:UILabel!
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        setupLabel(frame)
    }
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
    func setupLabel(frame:CGRect){
        self.label = UILabel(frame:frame)
        label.textColor = UIColor.grayColor()
        label.layer.borderWidth = 1;
        label.layer.borderColor = UIColor.darkGrayColor().CGColor
        self.contentView.addSubview(self.label)
    }
    
}
